lua-mosquitto.c

```diff
@@ -262,6 +262,33 @@ static int ctx_login_set(lua_State *L)
 	return mosq__pstatus(L, rc);
 }
 
+static int ctx_tls_opts_set(lua_State *L)
+{
+	ctx_t *ctx = ctx_check(L, 1);
+	int cert_reqs = luaL_optinteger(L, 2, 0);
```

Comment:
You're using a different default to mosquitto itself.  This should either default to the same as mosquitto does, or even be a required option, not an optional one.